appeals/e2e/cypress/e2e/back-office-appeals/manageDocsLpaq.spec.js

```diff
@@ -0,0 +1,64 @@
+// @ts-nocheck
+/// <reference types="cypress"/>
+
+import { users } from '../../fixtures/users';
+import { CaseDetailsPage } from '../../page_objects/caseDetailsPage.js';
+import { urlPaths } from '../../support/urlPaths.js';
+import { happyPathHelper } from '../../support/happyPathHelper.js';
+
+const caseDetailsPage = new CaseDetailsPage();
+
+describe('Remove doc from upload page', () => {
+	beforeEach(() => {
+		cy.login(users.appeals.caseAdmin);
+	});
+
+	it('add a doc and then remove on file upload page', () => {
+		cy.createCase().then((caseRef) => {
+			cy.addLpaqSubmissionToCase(caseRef);
+			happyPathHelper.assignCaseOfficer(caseRef);
+			happyPathHelper.reviewAppellantCase(caseRef);
+			happyPathHelper.startCase(caseRef);
+			caseDetailsPage.clickReviewLpaq();
+			caseDetailsPage.clickAddNotifyingParties();
+			caseDetailsPage.uploadSampleDoc();
+			caseDetailsPage.checkFileNameDisplays('sample-file.doc');
```

Comment:
Minor point, but ideally we should be getting the value of `sample-doc.doc` from within the `caseDetailsPage` as that's where the data originates from - i.e. it's a hard coded value within the `caseDetailsPage.uploadSampleDoc()` method. So near top of the file where you've got the `elements` collection for example you could add in

```
	sampleFiles = {
		document: 'sample-file.doc'
	}
```

You could also then upload the `uploadSampleDoc()` method to use this value also, so it would become something like

```

uploadSampleDoc() {
    cy.get('#upload-file-1').selectFile('cypress/fixtures/` + sampleFiles.document, { force: true });
 }
```

There are several other methods in `caseDetailsPage` that you could do the same thing for and move their sample file references into this collection. Finally it would then also make sense to make `cypress/fixtures/` into a constant and use that in all the methods also.

So this is all about using refactoring to improve the maintainability and structure of code, without changing the it's function. Ideally it's something you should look to do when you work on functional code so that you're gradually making it all more maintainable